github.pl

```diff
@@ -22,10 +22,17 @@
     merge_status(Dict.merged, Status),
     store_fact(Gen, github_pr_merged(Owner, Project, Pr, Status)).
 
+update_github_issues_facts(Gen) :-
+    config(github, [Owner, Project]),
+    github_issues(Owner, Project, Issues),
+    member(Issue, Issues),
+    store_fact(Gen, github_issue(Owner, Project, Issue.id, Issue.html_url, Issue.title, Issue.state, Issue.user.login)).
+
 merge_status(false, no).
 merge_status(true, yes).
 
 :- add_fact_updater(github:update_github_facts).
+:- add_fact_updater(github:update_github_issues_facts).
```

Comment:
why don't you call it github:update_github_facts ?

Reply:
done.